index.js

```diff
@@ -91,12 +91,12 @@ module.exports = function (filename, opts) {
     return offset - getOffsetInBlock(offset)
   }
 
-  function getBlockIndex(offset) {
-    return getBlockStart(offset) / blockSize
+  function getNextBlockStart(offset) {
+    return getBlockStart(offset) + blockSize
   }
 
-  function getNextBlockIndex(offset) {
-    return (getBlockIndex(offset) + 1) * blockSize
+  function getBlockIndex(offset) {
+    return getBlockStart(offset) / blockSize
   }
```

Comment:
The two diffs above look funky, but basically I just moved `getBlockIndex()` down, and moved `getNextBlockIndex()` up. Then renamed it to `getNextBlockStart()` and implemented it in a different way.